### ISSUE TITLE
[BUG] Seaport Trades Missing Trades

### ISSUE THREAD

**Issue by MSilb7** [duneanalytics/spellbook#5548]
### Description
Seaport Optimism spell is missing [this trade - via Sound Marketplace](https://optimistic.etherscan.io/tx/0x31572e6c00107d328295da2e03c61ba75ad509154778e197a1ebd324f9b28504) (seems to be the `matchAdvancedOrders` not being picked up.

### Current behavior
Transaction does not appear

### Expected behavior
Transaction appears

### Impacted model(s)

```
seaport_optimism.trades, nft.trades
```

### Possible solution
Hard to follow the spell logic, but seems like `matchAdvancedOrders` is not being picked up.

Discovered in https://github.com/duneanalytics/spellbook/pull/5547 - trying to add Base and Zora to the seaport models (also hard to do).

**Comment by jeff-dude** [comment]
closing issue due to inactivity. if still relevant, and proposed fix is available, feel free to reopen an issue as needed.
